class-ssas

NataliiaProtsanyk opened the pull request:
Задала стилі footer.
це не кінечний результат.

NataliiaProtsanyk committed:
class-ssas

Задала стилі footer.
це не кінечний результат.

Vasyl-Dzumedzei commented on src/sass/layout/_footer.scss:
```diff
@@ -1,4 +1,101 @@
+    color: $color-ellipse;
+    font-weight: normal;
+    text-transform: uppercase;
+@mixin font { 
```
міксіни прописуються так:  @include font(16px, 1.313, 0em, 700, center);

NataliiaProtsanyk replied:
Я просто не дуже тоді зрозуміла, виправлю.

Vasyl-Dzumedzei commented on src/sass/layout/_footer.scss:
```diff
@@ -1,4 +1,101 @@
+    line-height: 1.147;
+    letter-spacing: 0.04em;}
+}
+.social-networks-wrapper {
```
порожній селектор

NataliiaProtsanyk replied:
Ще не всі стилі задала всім селекторам.

Vasyl-Dzumedzei commented on src/sass/layout/_footer.scss:
```diff
@@ -1,4 +1,101 @@
+        margin-right: 28px;
+    }
+}
+.social-networks__link {
```
порожній селектор

Vasyl-Dzumedzei commented on src/sass/layout/_footer.scss:
```diff
@@ -1,4 +1,101 @@
+        background-color: $primary-white-color;
+    }
+}
+.address {
```
Порожній селектор

Vasyl-Dzumedzei reviewed (changes requested):
Повиправляй міксіни, бо шрифти виглядають трохи дивно на локал-хості та повидаляй порожні селектори

NataliiaProtsanyk committed:
class-ssas

Прописала медіа, але виникла проблема social-networks-wrapper, не можу нормально зпозиціонувати.
